app/assets/javascripts/components/menu_item.js.jsx

```diff
@@ -0,0 +1,19 @@
+var React = require('react');
+
+var MenuItem = React.createClass({
+
+  navigate: function(hash) {
+    window.location.hash = hash;
+  },
+
+  render: function() {
+    var classStyle = "menu-item";
+    if (this.props.hash === "share-facebook" || this.props.hash === "get-started") {
```

Comment:
No, the PR is automatically updated because it is actually tied to your branch's head, not a specific commit.